diff --git a/CMakeLists.txt b/CMakeLists.txt
index da38e41..7b061cb 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,8 +20,8 @@ install(EXPORT argparseConfig
 install(FILES ${CMAKE_CURRENT_LIST_DIR}/include/argparse/argparse.hpp
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/argparse)
 
-export(EXPORT argparseConfig
-        NAMESPACE argparse::)
+# export(EXPORT argparseConfig
+#         NAMESPACE argparse::)
 
 option(ARGPARSE_BUILD_EXAMPLES OFF)
 option(ARGPARSE_BUILD_TESTS OFF)
