diff --git a/docs/user-guide/a3-02-reference-capability-atoms.md b/docs/user-guide/a3-02-reference-capability-atoms.md
index 1cb7f5b..9a6eead 100644
--- a/docs/user-guide/a3-02-reference-capability-atoms.md
+++ b/docs/user-guide/a3-02-reference-capability-atoms.md
@@ -953,7 +953,7 @@ Compound Capabilities
 > Collection of shader stages
 
 `meshshading`
-> Capabilities required to use mesh shading features
+> Ccapabilities required to use mesh shading features
 
 `shadermemorycontrol_compute`
 > (gfx targets) Capabilities required to use memory barriers that only work for raytracing & compute shader stages
@@ -1204,7 +1204,7 @@ Compound Capabilities
 
 Other
 ----------------------
-*Capabilities that may be deprecated*
+*Capabilities which may be deprecated*
 
 `SPIRV_1_0`
 > Use `spirv_1_0` instead
diff --git a/source/compiler-core/slang-spirv-core-grammar.h b/source/compiler-core/slang-spirv-core-grammar.h
index e91caa7..b3db67b 100644
--- a/source/compiler-core/slang-spirv-core-grammar.h
+++ b/source/compiler-core/slang-spirv-core-grammar.h
@@ -6,7 +6,7 @@
 #include "../core/slang-string.h"
 
 #include <optional>
-#include <spirv/unified1/spirv.h>
+#include "../../external/spirv-headers/include/spirv/unified1/spirv.h"
 
 namespace Slang
 {
diff --git a/source/core/slang-hash.h b/source/core/slang-hash.h
index eee5887..a6fab55 100644
--- a/source/core/slang-hash.h
+++ b/source/core/slang-hash.h
@@ -4,7 +4,7 @@
 #include "slang-math.h"
 #include "slang.h"
 
-#include <ankerl/unordered_dense.h>
+#include "../external/unordered_dense/include/ankerl/unordered_dense.h"
 #include <cstring>
 #include <type_traits>
 
